exercises/practice/allergies/allergies_test.go

```diff
@@ -7,11 +7,13 @@ import (
 
 func TestAllergies(t *testing.T) {
 	for _, test := range listTests {
-		if actual := Allergies(test.score); !sameSliceElements(actual, test.expected) {
-			t.Fatalf("FAIL: Allergies(%d): expected %#v, actual %#v", test.score, test.expected, actual)
-		} else {
-			t.Logf("PASS: Allergic to %#v", test.expected)
-		}
+		t.Run(test.description, func(t *testing.T) {
+			if actual := Allergies(test.score); !sameSliceElements(actual, test.expected) {
+				t.Fatalf("FAIL: Allergies(%d): expected %#v, actual %#v", test.score, test.expected, actual)
```

Comment:
```suggestion
				t.Fatalf("Allergies(%d)=%#v, expected %#v", test.score, actual, test.expected)
```

Removed the `FAIL` string and made the print results be more in line with what we are doing in other exercises.